rewrite.go

```diff
@@ -82,21 +84,25 @@ func rewriteGoFile(name, qual string, paths []string) error {
 	if !changed {
 		return nil
 	}
-
+	var buffer bytes.Buffer
+	if err = printerConfig.Fprint(&buffer, fset, f); err != nil {
+		return err
+	}
+	fset = token.NewFileSet()
+	f, err = parser.ParseFile(fset, "", &buffer, parser.ParseComments)
```

Comment:
I changed to to use the original `name` again, but i'm not sure what effect here.